openedx/core/djangoapps/credit/api.py

```diff
@@ -322,6 +322,93 @@ def get_credit_requests_for_user(username):
     return CreditRequest.credit_requests_for_user(username)
```

Comment:
Are we planning to release this API on Open edX, or is it internal only? (If we publish it publicly, we should include documentation.)